docs/docs/tags.md

```diff
@@ -0,0 +1,3 @@
+# Tags index
```

Comment:
What do you think about merging this page with docs/docs/sample-scripts/index.md? That way, we don't introduce a new file that we don't show anywhere else, and when people come to the script samples section, they get an overview of the available scripts on a single page. This will also help us preserve the navigation on the left-hand side when browsing through the list of samples.